packages/kotti-ui/source/kotti-field-select/components/GenericSelectField.vue

```diff
@@ -237,11 +240,14 @@ export default defineComponent({
 			})),
 			inputRef,
 			isDropdownOpen,
+			localQuery,
```

Comment:
This shouldn’t be necessary to expose to the template as you didn’t add any new template code in this PR
```suggestion
```